File: public/server/lua/src/public.lua
return require("umd").define({
    "exports",
}, function(exports)

    function exports.main(input)
        -- According to the prefix of the id, the corresponding key is publicized, 
        -- and the unpublished key will not be synchronized to the client
        local retval = {}
        retval["player"]={
            "mBase",
            "mList",
            "mListSlave",
            "mDict",
        }
        retval["globalChat"]={
            "mChatList",
            "mChatID",
        }


        return retval
    end

end)

File: public/server/go/src/public.go
package src

import "github.com/lockval/go2plugin"

func Public(input *go2plugin.Input) map[string]any {
	// According to the prefix of the id, the corresponding key is publicized,
	// and the unpublished key will not be synchronized to the client
	return map[string]any{
		"player:": []string{
			"mBase",
			"mList",
			"mListSlave",
			"mDict",
		},
		"globalChat:": []string{
			"mChatList",
			"mChatID",
		},
	}
}
